config/routes.rb

```diff
@@ -3,5 +3,6 @@
   api_version(:module => "Api::V1", :header => {
     name: "Accept", :value => "application/vnd.providesk; version=1"}) do
       resources :sessions, only: :create
+      resources :departments
```

Comment:
there is only one method in departments controller, so avoid creating unnecessary routes